helm/cluster-cloud-director/templates/_ssh.tpl

```diff
@@ -0,0 +1,40 @@
+# The helper functions here can be called in templates and _helpers.tpl
+# This file should be self-sufficient. Don't call any functions from _helpers.tpl
+
+
+{{- define "sshFiles" -}}
+{{- if $.Values.ssh.trustedCaKeys -}}
+- path: /etc/ssh/trusted-user-ca-keys.pem
+  permissions: "0600"
+  content: |
+    {{- range $.Values.ssh.trustedCaKeys}}
+    {{.}}
+    {{- end }}
+- path: /etc/ssh/sshd_config
+  permissions: "0600"
+  content: |
+    {{- .Files.Get "files/etc/ssh/sshd_config" | nindent 4 }}
+{{- end -}}
+{{- end }}
+
+{{- define "sshPostKubeadmCommands" -}}
+- systemctl restart sshd
+{{- end -}}
+
+{{- define "sshUsers" -}}
+{{- if $.Values.ssh.users -}}
+users:
+{{- range $.Values.ssh.users }}
+- name: {{ .name }}
+{{- if .sudo }}
+  sudo: {{ .sudo }}
+{{- end }}
+{{- if .authorizedKeys }}
+  sshAuthorizedKeys:
+  {{- range .authorizedKeys }}
+  - {{ . }}
+  {{- end }}
+{{- end }}
```

Comment:
I wonder if we need to redefine all of that or we can simply swallow whole yaml from values and guard the format with the json schema?

Reply:
Example https://github.com/giantswarm/cluster-api-cleaner-cloud-director/pull/16/files#diff-ad864067edba460eada82f703f6d2ff812b770aac60aadccfca9699c9bb70a7aR40-R42

Reply:
Yeah. We can do it. There are 2 issues:
1. `sshAuthorizedKeys ` vs `authorizedKeys `. I need to rename the interface. Since this chart is not used so much right now, it is OK to rename a field.
2. There are other fields under the `users` field in the template. Do we want to support all of them?